skyrim-platform/src/platform_se/skyrim_platform/Win32Api.cpp

```diff
@@ -15,8 +19,26 @@ JsValue Win32Api::LoadUrl(const JsFunctionArguments& args)
   return JsValue::Undefined();
 }
 
-JsValue Win32Api::ExitProcess(const JsFunctionArguments& args)
+JsValue ExitProcess(const JsFunctionArguments& args)
 {
   std::exit(0);
   return JsValue::Undefined();
 }
+
+JsValue FileInfo(const JsFunctionArguments& args)
+{
+  auto path = static_cast<std::string>(args[1]);
+  for (char c : path) {
+    if (!(('A' <= c && c <= 'Z') || ('a' <= c && c <= 'z') || c == '.')) {
+      throw std::runtime_error("FileInfo: forbidden characters in path");
```

Comment:
What about `ccBGSSSE001-Fish.esm` which is standard? We better should just check for `../`

Reply:
It would allow absolute paths

Reply:
Well. What about `ccBGSSSE001-Fish.esm`? File names can contain `-` and other non-alphabet characters.
We better just check that file is loaded into the game (iterate game structures)

Reply:
> just check that file is loaded into the game (iterate game structures)

That would require duplicating TS logic from #951 here - no point in using TS to (almost) directly call Papyrus from there. It probably would be harder to implement it in C++.

Reply:
> What about `ccBGSSSE001-Fish.esm`? File names can contain `-` and other non-alphabet characters.

Better to just allow these characters. I'm not sure everything will work properly if we would allow more characters. (Well... could blacklist `/`, `\`, and probably `:`, but uhh...)

https://stackoverflow.com/a/5747088